config/database_test.go

```diff
@@ -467,12 +466,31 @@ func TestDatabaseStoreSet(t *testing.T) {
 
 		_, err = ds.Set(newCfg)
 		if assert.Error(t, err) {
-			assert.True(t, strings.HasPrefix(err.Error(), "failed to persist: failed to write to database"))
+			t.Log(err.Error())
```

Comment:
I'm not 100% sure if we need this `Log` call here, probably makes more sense as part of the last parameter of the `assert.True` with the textual description of the error. I only want to get the `err.Error()` message if the test fails. maybe something like:

```suggestion
assert.True(t, strings.HasPrefix(err.Error(), "failed to persist: failed to query active configuration"), "Unexpected error message: "+err.Error())
```